nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -498,11 +498,17 @@ def connect(self):
                 "Did not receive connection from board within "
                 + "specified timeout (%fs)." % self.connect_timeout
             )
+
         if self.recv_buffer[0] < 0.0:
             # Received a "terminate client" packet from the board, terminate the
             # Nengo simulation.
+            reason = ""
+            if self.recv_buffer[0] <= -20:
+                reason = "Unable to load FPGA driver!"
+            elif self.recv_buffer[0] <= -10:
+                reason = "Unable to acquire FPGA resource!"
```

Comment:
Just noticed this. Perhaps instead of "FPGA resource", we should specifically state "FPGA resource lock"? I dunno...

Reply:
Yea, I don't have a strong opinion either way, so I'll add this :+1: